README.md

```diff
@@ -101,6 +101,15 @@ Whenever your application loads, these variables will be available in `ENV`:
 config.fog_directory  = ENV['S3_BUCKET']
 ```
 
+Comments may be added to your file as such:
+
+```comments
+#here is a comment
+SECRET_KEY=YOURSECRETKEYGOESHERE # comment
+```
+
+Variable names may not contain the # symbol, however values can.
```

Comment:
This doesn't appear to be true. If I do `VARIABLE=foo#bar` I get the value `foo` and not bar.